tests/bench/harness.py

```
@@ -576,10 +677,16 @@ def cleanup(self) -> None:
             except Exception as e:
                 print(f"Error removing temp directory: {e}")
 
+        # Close cache
+        try:
+            asyncio.run(self.close_cache())
+        except Exception as e:
```

Ruff (0.14.1)

Do not catch blind exception: `Exception`

(BLE001)